titus-ext/k8s/README.md

```diff
@@ -0,0 +1,5 @@
+To run remote integration tests run:
+```bash
+$ export K8S_API_SERVER=<k8s_hostname>
+$ gw clean remoteIntegrationTest
```

Comment:
expand `gw`, that is your own alias for `./gradlew --with-some-flags`, right? If there are many flags that should be used, I'd recommend checking them into `./scripts/dev/`